web/impact/impact/tests/test_user_detail_view.py

```diff
@@ -628,6 +630,36 @@ def test_patch_user_with_no_profile_not_allowed_on_missing_fields(self):
             assert MISSING_PROFILE_ERROR.format(user.id) in response.data
             assert MISSING_SUBJECT_ERROR in response.data
 
+    def test_get_user_latest_active_program_location(self):
+        program = ProgramFactory(
+            program_status=ACTIVE_PROGRAM_STATUS,
+            program_family=ProgramFamilyFactory(),
```

Comment:
ProgramFamily will be provided by the ProgramFactory, only needs to be specified if you have a particular Family in mind